inc/classes/admin/settings/class-page.php

```diff
@@ -888,21 +888,57 @@ private function media_section() {
 			]
 		);
 
-		$rocket_maybe_disable_lazyload_plugins = [];
+		$disable_images_lazyload  = [];
+		$disable_iframes_lazyload = [];
+
+		if ( rocket_avada_maybe_disable_lazyload() ) {
+			$disable_images_lazyload[] = __( 'Avada', 'rocket' );
+		}
+
 		if ( rocket_maybe_disable_lazyload() ) {
-			$rocket_maybe_disable_lazyload_plugins[] = __( 'Autoptimize', 'rocket' );
+			$disable_images_lazyload[]  = __( 'Autoptimize', 'rocket' );
+			$disable_iframes_lazyload[] = __( 'Autoptimize', 'rocket' );
```

Comment:
```suggestion
```
I double-checked and Autoptimize doesn't have iframes lazyload, so this can be removed